app/components/IncentiveText.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss";
+@import "../../common/sass/variables.scss";
+
+.incentive {
+  padding-top: $space-bottom-between-elements;
+
+  h1 {
+      margin-top: $margin-bottom-between-paragraphs;
+  }
+  p {
+      margin-top: $margin-bottom-between-paragraphs;
+      @media (min-width : $screen-lg) {
+          padding-right: 160px;
+      }
+  }
+  a {
+    font-weight: $font-weight-more-bold;
+  }
+  button {
+      margin-top: $margin-bottom-between-paragraphs;
+      @media (max-width: $screen-sm) {
+          margin-left:auto;
+          margin-right:auto;
+          display:block;
```

Comment:
Space expected after `:` in line, 22, 23, 24.
I'm wondering how sass linter pass this.

Reply:
@krzkaczor @MostafaBalata It's because it didn't run. This branch doesn’t have it in code. I guess when it will be merged it will make master stop passing CI tests O_o.